docs/en/development/build-osx.md

```diff
@@ -38,7 +38,11 @@ $ cd ClickHouse
 ``` bash
 $ mkdir build
 $ cd build
-$ cmake .. -DCMAKE_CXX_COMPILER=`which clang++` -DCMAKE_C_COMPILER=`which clang`
+$ export PATH=/usr/local/opt/llvm/bin:$PATH
+$ export LDFLAGS="-L/usr/local/opt/llvm/lib"
+$ export CPPFLAGS="-I/usr/local/opt/llvm/include"
+$ LDFLAGS="-L/usr/local/opt/llvm/lib -Wl,-rpath,/usr/local/opt/llvm/lib"
```

Comment:
Why to add flags to external LLVM if we have our own? What issue does it resolve?

Reply:
+1

All build options should be controlled by CMake.